aspnetcore/blazor/call-web-api.md

```diff
@@ -236,6 +236,37 @@ You can address this by flowing prerendered state using the Persistent Component
 
 :::moniker-end
 
+:::moniker range=">= aspnetcore-9.0"
+
+## Client-side request streaming
+
+For Chromium-based browsers (for example, Google Chrome and Microsoft Edge) using the HTTP/2 protocol, HTTPS, and [CORS](xref:security/cors), client-side Blazor uses [Streams API](https://developer.mozilla.org/docs/Web/API/Streams_API) to permit [request streaming](https://developer.chrome.com/docs/capabilities/web-apis/fetch-streaming-requests).
```

Comment:
This makes it sound like CORS is always required for request streaming, even for same origin requests. Is this really correct? I suspect that CORS is only required for cross origin requests, just like with any other fetch request.

Reply:
```suggestion
For Chromium-based browsers (for example, Google Chrome and Microsoft Edge) using the HTTP/2 protocol, and HTTPS, client-side Blazor uses [Streams API](https://developer.mozilla.org/docs/Web/API/Streams_API) to permit [request streaming](https://developer.chrome.com/docs/capabilities/web-apis/fetch-streaming-requests).
```